classes/TaskRunner/Upgrade/UpgradeDb.php

```diff
@@ -68,7 +69,11 @@ public function getCoreUpgrader()
             return new CoreUpgrader17($this->container, $this->logger);
         }
 
-        return new CoreUpgrader80($this->container, $this->logger);
+        if (version_compare($this->container->getState()->getInstallVersion(), '8.1', '<')) {
+            return new CoreUpgrader80($this->container, $this->logger);
+        }
+
+        return new CoreUpgrader81($this->container, $this->logger);
```

Comment:
Aren't we going to need a CoreUpgrader90 then? 😋 so maybe we should prepare the code to welcome it?

My suggestion is a `switch` case:
- PS 8.0 ? Use CoreUpgrader80
- PS 8.1 ? Use CoreUpgrader81
- Other ? `throw new RuntimeException('Support for versions higher than 8.1 have not been implemented yet');`

Reply:
We don't actually know if we will need a CoreUpgrader90 yet :/

Reply:
OK for me 👍